scripts/tag_west_repos.sh

```diff
@@ -36,20 +36,27 @@
 
 declare -A PROJECT_TAGS
 
-# Uncomment and edit these lines as needed.
+# Set the tag names for all the projects below.
 #
 #     - keys are project names in west.yml
 #     - values are the tags you want to make
 #
-# PROJECT_TAGS[zephyr]=v2.7.99-ncs1-rc1
-# PROJECT_TAGS[mcuboot]=v1.8.99-ncs1-rc1
-# PROJECT_TAGS[trusted-firmware-m]=v1.5.0-ncs1-rc1
-# PROJECT_TAGS[matter]=v1.9.0-rc1
-# PROJECT_TAGS[openthread]=v1.9.0-rc1
-# PROJECT_TAGS[nrfxlib]=v1.9.0-rc1
-# PROJECT_TAGS[find-my]=v1.9.0-rc1
-# PROJECT_TAGS[nrf-802154]=v1.9.0-rc1
-# PROJECT_TAGS[homekit]=v1.9.0-rc1
+# nrfx/nRF/srcx repositories: vX.Y.Z(-rcN)
+PROJECT_TAGS[nrfxlib]=""
+PROJECT_TAGS[find-my]=""
+PROJECT_TAGS[homekit]=""
+PROJECT_TAGS[matter]=""
```

Comment:
Why isn't `matter` considered as OSS repo? It's a downstream of https://github.com/project-chip/connectedhomeip

Reply:
Actually, with the tagging scheme in the comment perhaps it'd be better to keep it here, even though it's OSS - we use NCS version to tag `matter` repo.

Reply:
It's a good question, and there's two answers: it's been managed by the Matter team and not the core Code Owners, and the upstream project makes no releases at all. I suggest we leave it here for now.